homeassistant/components/modbus/modbus.py

```diff
@@ -208,6 +205,14 @@ def setup(self):
         # Connect device
         self.connect()
 
+        # Start counting down to allow modbus requests.
+        if self._config_delay:
+            async_call_later(hass, self._config_delay, self.end_delay)
```

Comment:
Do we need to unsubscribe this listener on Home Assistant stop event?

Reply:
Good question, if HA starts and are stopped within a couple of seconds then yes.

I am not sure how to unsubscribe a "call_later", but I will check for a call to do that.

Reply:
Store the return value here and call it to unsubscribe.

Reply:
That was easy done as a security measure.